pkg/defaultinvariants/types.go

```diff
@@ -70,6 +71,7 @@ func newDefaultInvariants() invariants.InvariantRegistry {
 
 	invariantTests.AddInvariantOrDie("apiserver-availability", "kube-apiserver", disruptionlegacyapiservers.NewAvailabilityInvariant())
 
+	invariantTests.AddInvariantOrDie(disruptionpodnetwork.InvariantName, "Network / ovn-kubernetes", disruptionpodnetwork.NewPodNetworkAvalibilityInvariant())
```

Comment:
Nit: you use the const for the invariant name but not for the JIRA organizations name. Should it be consistent?